tests/integrationtest/r/privilege/privileges.result

```diff
@@ -509,7 +509,7 @@ GRANT ALL ON *.* TO 'superuser';
 SET PASSWORD for 'nobodyuser' = 'newpassword';
 SET PASSWORD for 'nobodyuser' = '';
 SET PASSWORD for 'superuser' = 'newpassword';
-Error 1044 (42000): Access denied for user 'superuser'@'%' to database 'mysql'
+Error 1044 (42000): Access denied for user 'nobodyuser'@'127.0.0.1' to database 'mysql'
```

Comment:
Looks like this is also testing the case we are described, so I changed the expected results here. Please let me know if my understanding is wrong: we are letting `nobodyuser` modify the password of `superuser`, hence the error message should say that `nobodyuser` has no access with its current host name.

@lcwangchao @xhebox